src/Components/Register/Register.tsx

```diff
@@ -60,7 +59,6 @@ const Register: React.FC = () => {
 
   return (
     <div className="auth-container container-fluid">
-    <ToastContainer />
```

Comment:
consider relocating the ToastContainer directly to the App. This enables the use of the toast function from any part of your application. By placing the ToastContainer at the top level, it becomes accessible to all child components of the App, streamlining its usage throughout the application.